### PR TITLE
Add the option to Hide the Frame area specific

### PR DESCRIPTION
**Pull request by fuba82**
hi there again ;)

i've added the option to hide the Frame area specific like this:
Raid (all raids not just Draenor)
5 Man Instance
Arena
Battleground
Scenario

hope there are no "whitespaces" this time ;)

### CODE REVIEW

**Comment by fuba82** [comment]
damn i forgor about the header localisation...
should be fixed now!

File: Bosses.lua [hunk 1]
```diff
@@ -228,15 +228,17 @@ end
...
-		if self.f.db.hideInRaid then
-			local _, _, _, _, _, _, _, instanceId = GetInstanceInfo()
-			if instanceId == 1228 or instanceId == 1205 or instanceId == 1448 then -- Highmaul, Blackrock Foundry, Hellfire Citadel
-				hidden = true
-				self.f:Hide()
-			elseif hidden then
-				hidden = false
-				self.f:Show()
-			end
+		if	(self.f.db.hideInRaid and (select(2, GetInstanceInfo()) == "raid")) or
```
Comment (funkydude) [inline Bosses.lua:231]: There's no need to call GetInstanceInfo 5 times.

File: LegionInvasionTimer.lua [hunk 2]
```diff
@@ -243,6 +243,11 @@ frame:SetScript("OnEvent", function(f)
 			colorComplete = {0,1,0,1},
 			colorIncomplete = {1,0,0,1},
 			colorBarBackground = {0,0,0,0.75},
+			hideInRaid = true,
```
Comment (funkydude) [inline LegionInvasionTimer.lua:246]: There entries are not needed, they shouldn't be on by default.

File: Options/Options.lua [hunk 3]
```diff
@@ -307,30 +307,91 @@ local acOptions = {
...
+			set = function(info, value)
+				lit.db.hideInBattleground = value
+				if (value) and (select(2, GetInstanceInfo()) == "pvp") then
+					lit:Hide();
+				elseif (not value) and (select(2, GetInstanceInfo()) == "pvp") then
+					lit:Show();
+				end
+			end,
+		},
+		hideInScenario  = {
```
Comment (funkydude) [inline Options/Options.lua:380]: There is no need for an option to hide in scenario, there are no relevant scenarios currently.

**Comment by fuba82** [comment]
i don't care anymore because the event is running out in few days ^^
use it or not it's your addon and just an idea ;)

edit:
Btw the Legion Questline -> Attack the Broken Shore is a Scenario ^^

**Pull request closed without merging**